vizro-core/changelog.d/20240109_183336_huong_li_nguyen_warning_bootstrap.md

```diff
@@ -0,0 +1,47 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Removed
+
+- A bullet item for the Removed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Added
+
+- A bullet item for the Added category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Changed
+
+- Add warning to docs about incompatibility with Dash Bootstrap Themes. ([#258](https://github.com/mckinsey/vizro/pull/258))
+
```

Comment:
```suggestion
```
I think that it would be overkill to add this to a changelog and it would unnecessarily crowd it. After all there was nothing wrong with the docs (e.g. a faulty example or so), but we are just improving them.

Reply:
Alright! Will remove